indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -58,7 +58,8 @@ def _serialize_availability(availability):
         data['nonbookable_periods'] = serialize_nonbookable_periods(data.get('nonbookable_periods', {}))
         data['unbookable_hours'] = serialize_unbookable_hours(data.get('unbookable_hours', {}))
         data.update({k: serialize_occurrences(data[k]) if k in data else {}
-                     for k in ['candidates', 'pre_bookings', 'bookings', 'conflicts', 'pre_conflicts']})
+                     for k in ['candidates', 'pre_bookings', 'bookings', 'conflicts', 'pre_conflicts',
```

Comment:
could use a tuple linstead of a list

Reply:
Hm, yeah, ok. However I really don't see a benefit to this ;P